### ISSUE TITLE
Large GRF option is unused

### ISSUE THREAD

**Issue by ThanatosShinji** [intel/llvm#13141]
### Describe the bug

I'm studying this topic of dpcpp guide: https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2024-0/small-register-mode-vs-large-register-mode.html
But I got this warning:
`icpx: warning: argument unused during compilation: '-Xsycl-target-backend -options -ze-opt-large-register-file' [-Wunused-command-line-argument]`

### To reproduce

```bash
icpx -fsycl -fsycl-targets=spir64 -Xsycl-target-backend "-options -ze-opt-large-register-file" -c test.cpp
``` 

### Environment

- OS Windows
- DPCPP  Intel® oneAPI Base Toolkit 2024.0.1

**Comment by asudarsa** [comment]
I think this is expected behavior. This flag does not mean anything during compilation to fat object (-c). This option is used to pass flags to backend compiler and this passing is done during AOT or offload wrapping stages, both of which happen during post-link stage.

Hope this answers the question.

Thanks

**Comment by ThanatosShinji** [comment]
Thanks for your reply! 
Does the executable project that links the fat object with this flag inherit its flag? Or this flag must be set to the executable project? I'm using it in a CMake project, what's the best practice to add this option in CMake?

Could you show me any method to print the register usage like nvcc does?
